scalatest.dotty/src/main/scala/org/scalatest/Expectations.scala

```diff
@@ -21,8 +21,14 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
 import scala.reflect.ClassTag
 
+import scala.tasty._
+import scala.quoted._
+
 private[scalatest] trait Expectations {
 
+  inline def (inline x: String) stripMargin <: String =
+    ${ Expectations.stripMarginImpl(x) }
+
```

Comment:
@cheeseng I'm not sure if this is the right place to put the extension method.

This compile-time `stripMargin` can co-exist with runtime `stripMargin` without problem, as the folllowing PR shows:

https://github.com/lampepfl/dotty/pull/6262